ChangeLog

```diff
@@ -1,3 +1,147 @@
+	  valid. (CFE-1915)
+	- Allow requiring TLS 1.3 as the minimum version
+	- Apply augments after vars, classes and inputs in def.json
+	  (CFE-2741, CFE-2844)
+	- Bundle name is now part of the log message when aborting a bundle
+	  (CFE-2793)
+	- Class names set by module protocol are automatically canonified
+	  (CFE-2877, CFE-2887)
+	- Classes failsafe_fallback and bootstrap_mode are now reported by default
+	- Correct log level for data_readstringarray* (CFE-2922)
+	- Do not iterate over JSON objects' properties in mustache (CFE-2125)
+	- Do not render templates when passed invalid data (CFE-2194)
+	- Eliminated error messages caused by attempting to kill expired processes
+	  (CFE-2824)
+	- Extend the test to cover more cases (ENT-3959)
+	- Fix cf-runalerts systemd unit conditions so the service will run
+	  (ENT-3929)
+	- Fix the off-by-one error in cf-runagent background process spawning
+	  (CFE-2873)
+	- Fixed OOB read / heap buffer overflow in evaluator (ENT-4136)
+	- Fixed a memory leak which occured when reloading RSA keys from disk
+	- Fixed a memory leak which occured while loading augments files
+	  (CFE-2913)
+	- Fixed an issue with splay time in cf-execd (CFE-2931)
+	- Fixed error handling and memory leak in cf-key (CFE-2918)
+	- Fixed memory leak in JSON to policy conversion (ENT-4136)
+	- Fixed memory leak in lmdb cleanup (CFE-2918)
+	- Fixed memory leaks in cf-agent during bootstrap (CFE-2918)
+	- Fixed memory leaks in variablesmatching() and findfiles() (CFE-2918)
+	- Fixed missing class with mustache templates in warn_only mode
+	  (CFE-2600)
+	- Fixed small memory leak in cf-serverd (CFE-2918)
+	- Fixed small memory leak in cf-upgrade (ENT-4136)
+	- Fixed small memory leaks of environment variable strings (CFE-2918)
+	- LMDB database dumper, lmdump, no longer creates empty databases
+	  (ENT-4064)
+	- Made variablesmatching functions treat args regexes more correctly
+	  variablesmatching() and variablesmatching_as_data() no longer
+	  use string comparison to find matches. The documentation is clear;
+	  arguments should be regexes (so you have to escape special
+	  characters).
+	  bundle agent main
+	  {
+	      vars:
+	          "myvar"
+	              string => "example",
+	              meta => {"os[linux]"};
+	          "matches"
+	              slist => variablesmatching(".*", "os\[linux\]");
+	      reports:
+	          "Match: $(matches)";
+	  }
+	  The above example is correct. If you don't escape the brackets
+	  like above, it will no longer work. (You probably shouldn't use
+	  brackets in tags anyway).
+	- Prevent the init script from managing processes inside containers
+	  (ENT-3800)
+	- Read mustache-rendered files in text mode when comparing digest
+	  (ENT-2526)
+	- Reload persistent classes on config reload in cf-execd and cf-serverd
+	  (CFE-2857)
+	- Test that classes can be defined without constraints (CFE-2000)
+	- Trigger promises.cf as the last step of bootstrap (CFE-2826)
```

Comment:
Move this up to the entry about `--skip-bootstrap-policy-run` which is referencing this feature.